regression/upgrade/upgrade2.0.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+
+UpgradeFrom=$1
+UpgradeTo=$2
+CurrentDirectory=$(cd `dirname $0` && pwd)
+FabricTestDir="$(echo $CurrentDirectory | awk -F'/fabric-test/' '{print $1}')/fabric-test"
+OperatorDir="$FabricTestDir"/tools/operator
+testdataDir="$FabricTestDir"/regression/testdata
+
+fabrictestCheckout(){
+    echo "Checking out to $1 branch"
+    git checkout -- .
+    rm go.sum
```

Comment:
@btl5037 can you give your insights about this removing go.sum and then checking out from 1.4 to master

Reply:
We need to either, fix the go.mod in the root of the repo or add a go.mod to the operator directory. I'm going to go for number 2, let me see if I can get that right

Reply:
ok

Reply:
If you rebase now, you should get regenerated go.sum's